[2024-11-06T15:14:31.647+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-06T15:14:32.080+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_V3.get_ageTask scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-06T15:14:32.520+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_V3.get_ageTask scheduled__2024-11-04T00:00:00+00:00 [queued]>
[2024-11-06T15:14:32.532+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-06T15:14:32.844+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_ageTask> on 2024-11-04 00:00:00+00:00
[2024-11-06T15:14:32.928+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=214) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-06T15:14:32.966+0000] {standard_task_runner.py:72} INFO - Started process 237 to run task
[2024-11-06T15:14:32.951+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_V3', 'get_ageTask', 'scheduled__2024-11-04T00:00:00+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python.py', '--cfg-path', '/tmp/tmp71c6q4ra']
[2024-11-06T15:14:33.046+0000] {standard_task_runner.py:105} INFO - Job 184: Subtask get_ageTask
[2024-11-06T15:14:33.674+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_V3.get_ageTask scheduled__2024-11-04T00:00:00+00:00 [running]> on host b0b6b56c3157
[2024-11-06T15:14:35.406+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_V3' AIRFLOW_CTX_TASK_ID='get_ageTask' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T00:00:00+00:00'
[2024-11-06T15:14:35.407+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-06T15:14:35.988+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-06T15:14:36.045+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-06T15:14:36.050+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_V3, task_id=get_ageTask, run_id=scheduled__2024-11-04T00:00:00+00:00, execution_date=20241104T000000, start_date=20241106T151432, end_date=20241106T151436
[2024-11-06T15:14:36.585+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-06T15:14:37.054+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-06T15:14:37.490+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
